ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -5,6 +5,7 @@
     catalog_location={{ item.value.location }}
     catalog_repo_url={{ item.value.url }}
     api_host={{ groups['edge'] | first }}
+    whisk_api_host={{ whisk_api_host_name | groups['edge'] | first }}
```

Comment:
IIRC actions should receive this info via env variable `__OW_API_HOST`. If there's an action inside the catalog that doesn't make use of this we could open an issue there ?

Reply:
looking at the changes in https://github.com/apache/incubator-openwhisk-catalog/pull/243 I'm reading that this is needed only for installing the catalog ? I'm confused b/c the catalog installs correctly on the local env.

Reply:
@ddragosd I am working on changing this such that it can use the already existing variable instead of creating a new one. Will update accordingly.